core/datainput.gms

```diff
@@ -462,6 +462,11 @@ $offdelim
 $Onlisting
 $ENDIF.WindOff
 
+*** test-add USA cap. Forecast by Wood Mackenzie Solar Market Insight Report 2022 ~ 265 GW DC in 2025 in bullish scenario
+!! Should be less in GW_AC, but REMIND corrects for lower model CF than real world by upscaling capacity - should be roughly ok.
+pm_histCap("2025","USA","spv") = 0.265;
```

Comment:
this part was only for test, right? should be deleted also as it includes a specific region

Reply:
yes, totally - this was only there to be able to test it before the new input data is available. 
will delete